### ISSUE TITLE
Is this still working?

### ISSUE THREAD

**Issue by calexandre** [vookimedlo/OctoPrint-Prusa-Mini-ETA#6]
I just acquired a Prusa Mini and wanted to try this plugin... But my octoprint still does not show the ETA...

Using `PrusaSlicer 2.3.0` + `OctoPrint 1.5.3` + `OctoPi 0.17.0`

![image](https://user-images.githubusercontent.com/1485193/107973158-c5e35480-6fac-11eb-842f-cfb7d06ba5e6.png)

**Comment by Dario503** [comment]
Not working for me either.  Using Latest Octoprint v1.6.0 and v4.3.1 PRusa Mini+ firmware.  I just get a dot.

**Comment by mtaumike** [comment]
Yep, it's all broke now it seems though I bet the issue is upgraded firmware over time returning a pattern change
